test/e2e/common/networking.go

```diff
@@ -126,5 +126,21 @@ var _ = ginkgo.Describe("[sig-network] Networking", func() {
 				}
 			}
 		})
+
+		ginkgo.It("should function for pod-pod communication: sctp [LinuxOnly] [NodeConformance]", func() {
+			config := e2enetwork.NewNetworkingTestConfig(f, e2enetwork.EnableSCTP)
+			checkNodeConnectivity(config, "sctp", e2enetwork.EndpointSCTPPort)
```

Comment:
Why is pod-pod test checking node connectivity?

Reply:
I think we may have gotten the test names mixed up here

Reply:
I had to look it several times, but it actually tests pod to pod

Reply:
okay, I see it now

Reply:
There are fairly few callers of this function, can we rename it now?

Reply:
fair enough

Reply:
done